tests/integration_tests/test_mlflow.py

```diff
@@ -132,3 +134,31 @@ def test_tag_truncation(tmpdir: py.path.local) -> None:
 
     my_user_attr = first_run_dict["data"]["tags"]["my_user_attr"]
     assert len(my_user_attr) <= 5000
+
+
+def test_nest_trials(tmpdir: py.path.local) -> None:
+    tmp_tracking_uri = "file:{}".format(tmpdir)
+
+    study_name = "my_study"
+    mlflow.set_tracking_uri(tmp_tracking_uri)
+    mlflow.set_experiment(study_name)
+
+    mlflc = MLflowCallback(tracking_uri=tmp_tracking_uri, nest_trials=True)
```

Comment:
We might want to check the negative case where the parameter is `False`.

Reply:
If we set `nest_trials` to `False` and an active run exists, MLflow would raise the following error:

```
Exception: Run with UUID aa67b134da7c43158015648fc00ffdb4 is already active.
To start a new run, first end the current run with mlflow.end_run().
To start a nested run call start_run with nested=True
```

Should I add a test to verify this behavior?

Reply:
Yup, that' what I meant. Sorry I was probably not clear enough.